src/components/Inputs/Technoeconomic/TechnoeconomicInputs.tsx

```diff
@@ -25,7 +19,34 @@ interface Props {
   disabled: boolean;
 }
 
+const determineScaledCapitalCost = (electricalCapacity: number) => {
+  const scaleFactor = 0.82; // source: https://evcvaluation.com/cost-to-capacity-method-applications-and-considerations/
+  const defaultCapitalCost = 70_000_000;
+  const defaultElectricCapacity = 25_000;
```

Comment:
Can we write numbers like this, 70000000 as 70_000_000 and 25000 as 25_000?

Reply:
Yea just makes it easier to read.